api-lambda-ses/Readme.md

```diff
@@ -0,0 +1,63 @@
+# Integration of Amazon API Gateway REST API with AWS Lambda and Amazon SES
+
+The SAM template deploys a API Gateway REST API with Lambda function integration and the IAM permissions required to run the application. Whenever the REST API is invoked, the Lambda function publishes a message to the SES. 
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+* [Verified email addresses in SES](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html#just-verify-email-proc)
+
+## Deployment Instructions
+
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ``` 
+    
+2. Change directory to the pattern directory:
+    ``` 
+    cd apigw-lambda-ses
```

Comment:
This isn't your project directory and is what caused my errors. Yours is api-lambda-ses.